docker/utils/utils.py

```diff
@@ -952,6 +953,8 @@ def format_env(key, value):
         return '{key}={value}'.format(key=key, value=value)
     return [format_env(*var) for var in six.iteritems(environment)]
 
+def duration_from_string(duration_string):
+    return timeparse(duration_string) * 1000000000
```

Comment:
Should I make it so you just pass in the raw `{ Test, Interval, Timeout, Retries }` object then?
The consumer could then be responsible for converting the duration string into a duration if necessary, and docker-compose could do the conversion from the cleaner `{ command, interval, timeout, retries }` object then.